types/office-js-preview/index.d.ts

```diff
@@ -13936,11 +13936,16 @@ declare namespace Office {
         /**
          * Gets a string that contains a token used to call REST APIs or Exchange Web Services.
          *
-         * The getCallbackTokenAsync method makes an asynchronous call to get an opaque token from the Exchange Server that hosts the user's mailbox. 
+         * The `getCallbackTokenAsync` method makes an asynchronous call to get an opaque token from the Exchange Server that hosts the user's mailbox. 
          * The lifetime of the callback token is 5 minutes.
          *
          * The token is returned as a string in the `asyncResult.value` property.
          *
+         * To call the `getCallbackTokenAsync` method in read mode, **ReadItem** is the minimum permission level your app can specify in its manifest.
+         * 
+         * If the item has not yet been saved in compose mode, you must call the `saveAsync` method before calling the `getCallbackTokenAsync` method.
+         * To call the `saveAsync` method, **ReadWriteItem** is the minimum permission level your app can specify.
```

Comment:
Consider simplifying this to something like:
"Calling `getCallbackTokenAsync` in compose mode requires you to save the item. The `saveAsync` method requires a minimum permission level of **ReadWriteItem**."